### DISCUSSION TITLE
Add Audio editing software list

### DISCUSSION THREAD

**Discussion by Kapot** [pluja/awesome-privacy#294] [Ideas]
There is no audio editing software in the list, might i suggest:

- **Audicity** (https://github.com/audacity/audacity)
Free, open source, cross-platform audio software.
Audacity is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems.
Audacity is free, open source software.

- **Audiomass** (https://github.com/pkalogiros/AudioMass)
AudioMass is a free, open source, web-based Audio and Waveform Editor.
It runs entirely in the browser with no backend and no plugins required!

- **Mixxx** (https://github.com/mixxxdj/mixxx)
Mixxx is Free DJ software that gives you everything you need to perform live DJ mixes. Mixxx works on GNU/Linux, Windows, and macOS.

- **Qtractor** (https://github.com/rncbc/qtractor)
Qtractor is an Audio/MIDI multi-track sequencer application written in C++ with the Qt framework. Target platform is Linux, where the Jack Audio Connection Kit (JACK) for audio, and the Advanced Linux Sound Architecture (ALSA) for MIDI, are the main infrastructures to evolve as a fairly-featured Linux desktop audio workstation GUI, specially dedicated to the personal home-studio.
Qtractor is free, open-source software, distributed under the terms of the GNU General Public License (GPL) version 2 or later.

```
- [Audicity](https://github.com/audacity/audacity) - Free, open source, cross-platform audio software.
- [Audiomass](https://github.com/pkalogiros/AudioMass) - AudioMass is a free, open source, web-based Audio and Waveform Editor.
- [Mixxx](https://github.com/mixxxdj/mixxx) - Mixxx is Free DJ software that gives you everything you need to perform live DJ mixes.
- [Qtractor](https://github.com/rncbc/qtractor) - Qtractor is an Audio/MIDI multi-track sequencer application written in C++ with the Qt framework.
```

**Comment by Kapot** [comment]
Let me know if this isn't the way to suggest stuff, I'll move/edit with more info if needed

**Reply by pluja** [reply]
Yes, that's fine. You can also open a Pull Request if you are ok with that! That would be helpful

**Comment by pluja** [comment]
Hey, thank you a lot. Will take a look to these services/apps and be adding them if they are ok!
